Gemfile

```diff
@@ -3,6 +3,7 @@ source 'https://rubygems.org'
 gemspec
 
 group :development do
+  gem "rdoc"
```

Comment:
Could you keep this list in alphabetical order?

Reply:
Done.